2021-12-14 17:07:11,460 INFO    MainThread:27215 [wandb_setup.py:_flush():69] setting env: {'project': 'exercise_14', 'run_group': 'dev'}
2021-12-14 17:07:11,460 INFO    MainThread:27215 [wandb_setup.py:_flush():69] setting login settings: {}
2021-12-14 17:07:11,460 INFO    MainThread:27215 [wandb_init.py:_log_setup():334] Logging user logs to /Users/nicholas.osborne/Documents/python/ML_Dev_Ops/genre_classification/segregate/wandb/run-20211214_170711-1s17isvj/logs/debug.log
2021-12-14 17:07:11,460 INFO    MainThread:27215 [wandb_init.py:_log_setup():335] Logging internal logs to /Users/nicholas.osborne/Documents/python/ML_Dev_Ops/genre_classification/segregate/wandb/run-20211214_170711-1s17isvj/logs/debug-internal.log
2021-12-14 17:07:11,461 INFO    MainThread:27215 [wandb_init.py:init():367] calling init triggers
2021-12-14 17:07:11,461 INFO    MainThread:27215 [wandb_init.py:init():372] wandb.init called with sweep_config: {}
config: {}
2021-12-14 17:07:11,461 INFO    MainThread:27215 [wandb_init.py:init():416] starting backend
2021-12-14 17:07:11,461 INFO    MainThread:27215 [backend.py:_multiprocessing_setup():69] multiprocessing start_methods=spawn,fork,forkserver, using: spawn
2021-12-14 17:07:11,471 INFO    MainThread:27215 [backend.py:ensure_launched():123] starting backend process...
2021-12-14 17:07:11,478 INFO    MainThread:27215 [backend.py:ensure_launched():127] started backend process with pid: 27232
2021-12-14 17:07:11,481 INFO    MainThread:27215 [wandb_init.py:init():421] backend started and connected
2021-12-14 17:07:11,484 INFO    MainThread:27215 [wandb_init.py:init():461] updated telemetry
2021-12-14 17:07:11,486 INFO    MainThread:27215 [wandb_init.py:init():480] communicating current version
2021-12-14 17:07:12,166 INFO    MainThread:27215 [wandb_init.py:init():485] got version response upgrade_message: "wandb version 0.12.7 is available!  To upgrade, please run:\n $ pip install wandb --upgrade"

2021-12-14 17:07:12,166 INFO    MainThread:27215 [wandb_init.py:init():493] communicating run to backend with 30 second timeout
2021-12-14 17:07:12,681 INFO    MainThread:27215 [wandb_init.py:init():518] starting run threads in backend
2021-12-14 17:07:12,729 INFO    MainThread:27215 [wandb_run.py:_console_start():1476] atexit reg
2021-12-14 17:07:12,730 INFO    MainThread:27215 [wandb_run.py:_redirect():1339] redirect: SettingsConsole.REDIRECT
2021-12-14 17:07:12,731 INFO    MainThread:27215 [wandb_run.py:_redirect():1344] Redirecting console.
2021-12-14 17:07:12,731 INFO    MainThread:27215 [redirect.py:install():213] install start
2021-12-14 17:07:12,731 INFO    MainThread:27215 [redirect.py:install():228] install stop
2021-12-14 17:07:12,731 INFO    MainThread:27215 [redirect.py:install():213] install start
2021-12-14 17:07:12,732 INFO    MainThread:27215 [redirect.py:install():228] install stop
2021-12-14 17:07:12,732 INFO    MainThread:27215 [wandb_run.py:_redirect():1390] Redirects installed.
2021-12-14 17:07:12,732 INFO    MainThread:27215 [wandb_init.py:init():542] run started, returning control to user process
2021-12-14 17:07:20,610 INFO    MainThread:27215 [wandb_run.py:_atexit_cleanup():1446] got exitcode: 0
2021-12-14 17:07:20,610 INFO    MainThread:27215 [wandb_run.py:_restore():1418] restore
2021-12-14 17:07:20,611 INFO    MainThread:27215 [redirect.py:uninstall():232] uninstall start
2021-12-14 17:07:20,611 INFO    MainThread:27215 [redirect.py:_stop():287] _stop: stdout
2021-12-14 17:07:20,611 INFO    MainThread:27215 [redirect.py:_stop():293] _stop closed: stdout
2021-12-14 17:07:20,611 INFO    stdout    :27215 [redirect.py:_pipe_relay():129] relay done saw last write: stdout
2021-12-14 17:07:20,611 INFO    stdout    :27215 [redirect.py:_pipe_relay():145] relay done done: stdout
2021-12-14 17:07:20,611 INFO    MainThread:27215 [redirect.py:_stop():299] _stop joined: stdout
2021-12-14 17:07:20,611 INFO    MainThread:27215 [redirect.py:_stop():301] _stop rd closed: stdout
2021-12-14 17:07:20,611 INFO    MainThread:27215 [redirect.py:uninstall():236] uninstall done
2021-12-14 17:07:20,611 INFO    MainThread:27215 [redirect.py:uninstall():232] uninstall start
2021-12-14 17:07:20,611 INFO    MainThread:27215 [redirect.py:_stop():287] _stop: stderr
2021-12-14 17:07:20,611 INFO    MainThread:27215 [redirect.py:_stop():293] _stop closed: stderr
2021-12-14 17:07:20,612 INFO    stderr    :27215 [redirect.py:_pipe_relay():129] relay done saw last write: stderr
2021-12-14 17:07:20,612 INFO    stderr    :27215 [redirect.py:_pipe_relay():145] relay done done: stderr
2021-12-14 17:07:20,612 INFO    MainThread:27215 [redirect.py:_stop():299] _stop joined: stderr
2021-12-14 17:07:20,612 INFO    MainThread:27215 [redirect.py:_stop():301] _stop rd closed: stderr
2021-12-14 17:07:20,612 INFO    MainThread:27215 [redirect.py:uninstall():236] uninstall done
2021-12-14 17:07:21,091 INFO    MainThread:27215 [wandb_run.py:_wait_for_finish():1568] got exit ret: file_counts {
  wandb_count: 1
}
pusher_stats {
  uploaded_bytes: 1175
  total_bytes: 1175
}

2021-12-14 17:07:23,097 INFO    MainThread:27215 [wandb_run.py:_wait_for_finish():1568] got exit ret: file_counts {
  wandb_count: 5
}
pusher_stats {
  uploaded_bytes: 1719
  total_bytes: 2334
}

2021-12-14 17:07:25,100 INFO    MainThread:27215 [wandb_run.py:_wait_for_finish():1568] got exit ret: done: true
exit_result {
}
file_counts {
  wandb_count: 5
}
pusher_stats {
  uploaded_bytes: 2334
  total_bytes: 2334
}

2021-12-14 17:07:26,196 INFO    MainThread:27215 [wandb_run.py:_show_files():1790] logging synced files
